CRM/ManageLetterheads/Uninstall/DeleteNavigationItem.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * Uninstall Class for deleting the Letterhead navigation item.
+ */
+class CRM_ManageLetterheads_Uninstall_DeleteNavigationItem {
+
+  /**
+   * Used for running the uninstall sequence.
+   *
+   * @return bool
+   *   TRUE when the uninstall sequence runs successfully.
+   */
+  public function apply() {
+    $this->deleteManageLetterheadNavigationItem();
+
+    return TRUE;
+  }
+
+  /**
+   * Deletes the letterhead navigation item if it exists.
+   */
+  private function deleteManageLetterheadNavigationItem() {
+    $letterheads = civicrm_api3('Navigation', 'get', [
+      'sequential' => '1',
+      'name' => 'Letterheads',
+    ]);
+
+    if (!empty($letterheads['count'])) {
+      $navigationItemRecord = CRM_Utils_Array::first($letterheads['values']);
```

Comment:
instead of this can we apply limit of 1 to query/api call?

Reply:
Even if we apply a limit we need to get the first result from `$letterheads['values']`.

The other alternative would be to use `getsingle` instead of `get`, but we'd need to wrap this block in a try catch since `getsingle` throws an error when no records are found.